common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -742,16 +742,16 @@ func newShardPersistence(
 	}, nil
 }
 
-// NewWorkflowExecutionPersistence is used to create an instance of workflowExecutionManager implementation
-func NewWorkflowExecutionPersistence(
+// NewWorkflowStore is used to create an instance of workflowExecutionManager implementation
+func NewWorkflowStore(
 	shardID int32,
```

Comment:
can we remove this shard ID as input and put shard ID as part of persistence request?

Reply:
Agreed. But that itself is a bigger change. Not all persistence request has shard ID, and there are lots of logic build around execution store per shard ID.

Reply:
as long as we plan to do the ^ before 1.12.x